hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/sink/clustering/ClusteringCommitSink.java

```diff
@@ -97,6 +97,11 @@ public void open(Configuration parameters) throws Exception {
   @Override
   public void invoke(ClusteringCommitEvent event, Context context) throws Exception {
     final String instant = event.getInstant();
+    if (event.isFailed() || event.getWriteStatuses().stream().anyMatch(writeStatus -> writeStatus.getTotalErrorRecords() > 0)) {
+      LOG.warn("Receive abnormal ClusteringCommitEvent of instant " + instant + ", task ID is " + event.getTaskID()
+              + ", is failed: " + event.isFailed() + ", error record count: "
```

Comment:
I guess the condition `event.isFailed()` is also verbose and unnecessary because when it's true, very probablily an exception throwned with empty write statuses.

Reply:
If "an exception throwned with empty write statuses" the second condition `event.getWriteStatuses().stream().anyMatch(writeStatus -> writeStatus.getTotalErrorRecords() > 0)` will be false, 
 thus in this scene we will need the condition `event.isFailed()` to trigger the log, so seems like we need the condition `event.isFailed()` here?